terratorch/models/backbones/clay_v1/embedder.py

```diff
@@ -0,0 +1,173 @@
+import re
+import warnings
+
+import numpy as np
+import torch
+from torch import nn, Tensor
+import torch
+from timm.models import FeatureInfo
+from timm.models._builder import build_model_with_cfg
+from timm.models._registry import generate_default_cfgs, register_model
+
+from terratorch.models.backbones.clay_v1.modules import EmbeddingEncoder, Datacuber
+
+warnings.filterwarnings("ignore", category=UserWarning)
+
+
+default_cfgs = generate_default_cfgs(
+    {
+        "clay_v1_base": {
+            "hf_hub_id": "made-with-clay/Clay",
+            "hf_hub_filename": "clay-v1-base.ckpt"
+        }
+    }
+)
+
+
+class Embedder(nn.Module):
+    default_out_indices = (0,)  # Single out_indices for simplicity
+
+    def __init__(self,
+                 img_size=256,
+                 num_frames=1,
+                 ckpt_path=None,
+                 device="cuda",
```

Comment:
Device shouldn't be required to be passed. In pytorch lightning, `.to(device)` statements should not be used, as the responsibility to place modules in the correct device is passed to lightning.